src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
here should be also modifiers for calendar block.
something like that
```suggestion
    <div class="calendar calendar--start-day-wed calendar--month-length-30">
```